e2e/pages/events/manageDocumentsEvent.page.js

```diff
@@ -2,86 +2,69 @@ const { I } = inject();
+    supportingDocuments: function(index) {
+      return supportingDocumentsFragment.supportingDocuments(index, 'supportingEvidenceDocumentsTemp');
+    },
   },
 
   async selectFurtherEvidence() {
-    const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    I.click(this.fields(elementIndex, type).documentType.furtherEvidence);
+    I.click(this.fields.documentType.furtherEvidence);
   },
 
   async selectCorrespondence() {
-    const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    I.click(this.fields(elementIndex, type).documentType.correspondence);
+    I.click(this.fields.documentType.correspondence);
   },
 
   async selectC2SupportingDocuments() {
-    const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    I.click(this.fields(elementIndex, type).documentType.c2);
+    I.click(this.fields.documentType.c2);
   },
 
   async selectFurtherEvidenceIsRelatedToHearing() {
-    const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    I.waitForElement(this.fields(elementIndex, type).relatedToHearing.yes);
-    I.click(this.fields(elementIndex, type).relatedToHearing.yes);
+    I.waitForElement(this.fields.relatedToHearing.yes);
+    I.click(this.fields.relatedToHearing.yes);
   },
 
   async selectHearing(hearingDate) {
-    const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    I.waitForElement(this.fields(elementIndex, type).hearingList);
-    I.selectOption(this.fields(elementIndex, type).hearingList, `Case management hearing, ${hearingDate}`);
+    I.waitForElement(this.fields.hearingList);
+    I.selectOption(this.fields.hearingList, `Case management hearing, ${hearingDate}`);
   },
 
   async select2FromDropdown() {
-    const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    const dropdownLabel = await I.grabTextFrom(`${this.fields(elementIndex, type).c2DocumentsList} option:nth-child(2)`);
-    I.waitForElement(this.fields(elementIndex, type).c2DocumentsList);
-    I.selectOption(this.fields(elementIndex, type).c2DocumentsList, dropdownLabel);
+    const dropdownLabel = await I.grabTextFrom(`${this.fields.c2DocumentsList} option:nth-child(2)`);
+    I.waitForElement(this.fields.c2DocumentsList);
+    I.selectOption(this.fields.c2DocumentsList, dropdownLabel);
   },
 
   async enterDocumentName(documentName) {
     const elementIndex = await this.getActiveElementIndex();
-    const type = this.getSupportingEvidenceDocumentType();
-    I.fillField(this.fields(elementIndex, type).supportingDocuments.name, documentName);
+    I.fillField(this.fields(elementIndex).supportingDocuments.name, documentName);
```

Comment:
need to update this and the ones below to use index on supportingDocuments instead of fields

Reply:
ah yes ofc